WordPress/gutenberg
# Build separate full contributors list

## Description

In https://github.com/WordPress/gutenberg/issues/38749 we learned of a few reasons why listing the usernames of users next to their respective PRs might not be a great idea.

This PR changes things up so we have a single list of contributors underneath a `Contributors` section. It looks like this for `12.5`

```
## Contributors

The following users merged PRs in this release:

@aaronrobertshaw @adamziel @ajlende @alanjacobmathew @alexstine @amustaque97 @andrewserong @annezazu @anton-vlasenko @arthur791004 @audrasjb @c4rl0sbr4v0 @carolinan @chad1008 @ciampo @cr0ybot @dcalhoun @dmsnell @fabiankaegy @fluiddot @fullofcaffeine @geriux @getdave @glendaviesnz @guarani @gziolo @hypest @imangm @jameskoster @jasmussen @jeremyfelt @jhnstn @johnbillion @jorgecontreras @jorgefilipecosta @kebbet @kevin940726 @KevinBatdorf @knilkantha @Mamaduka @mchowning @mkevins @ndiego @noisysocks @ntsekouras @oandregal @ocean90 @ockham @peterwilsoncc @ramonjd @ribaricplusplus @richtabor @ryelle @scruffian @SiobhyB @talldan @tellthemachines @vcanales @walbo @youknowriad
```

**Note**: I'm not sure if this will cause GH to auto-generate it's own list thus duplicating the list. @priethor this is one of the reasons why I avoided this approach in my previous PR.

Closes https://github.com/WordPress/gutenberg/issues/38749

## Testing Instructions

- Run `npm run changelog -- --milestone="Gutenberg 12.6` in Gutenberg repo root.
- See generated changelog.
- `First time contributors` should be "as was"
- Check `Contributors` section is there and includes a single entry per username.
- Check not "bot" users are listed in either `First time contributors` or `Contributors` sections.

## Screenshots 

## Types of changes





## Checklist:
